db/schema.rb

```diff
@@ -32,9 +32,11 @@
     t.datetime "created_at"
     t.datetime "updated_at"
     t.string   "role"
+    t.integer  "user_id"
```

Comment:
Ok. So you never want to edit (or remove) a migration file that's already
been run. At this point, you're probably going to need to drop your
database and then recreate it. If your app was live and we had real
production data, we'd take a different approach so you don't lose all your
data, but since it's just in development, this is going to be the easiest
approach:

rake db:drop
rake db:create
rake db:migrate

That should get everything back to a functional state. If it doesn't let me
know. Or, if you prefer not to drop your database entirely, let me know and
we can try the other approach together.
On Fri, Jul 3, 2015 at 4:28 PM Brigid Prinsloo <notifications@github.com>
wrote:

> In db/schema.rb
> <https://github.com/the-b-boss/blocitoff/pull/3#discussion_r33881173>:
>
> > @@ -32,9 +32,11 @@
> >      t.datetime "created_at"
> >      t.datetime "updated_at"
> >      t.string   "role"
> > +    t.integer  "user_id"
>
> I had a bit of a blonde moment - not sure what I was thinking and realized
> it was a mistake but now I don't know how to undo it. I tried to delete the
> migration file and run a db migration again but it didn't work. I also
> tried to rollback using the terminal and it didn't work either. Just don't
> want it to mess stuff up further down the line...
> … <#msg-f:1505712037674293360_>
> Sent from my iPhone
>  On 03 Jul 2015, at 21:22, bjblock <notifications@github.com> wrote: In
> db/schema.rb: > @@ -32,9 +32,11 @@ > t.datetime "created_at" > t.datetime
> "updated_at" > t.string "role" > + t.integer "user_id" every table has an
> id column (known as the "primary key"), which does not need to be indexed.
> A user_id looks more like a "foreign key" which would be on another table
> referencing the users table. For example, in the last project, the Posts
> table had a user_id column which is how a post was associated to (or
> belongs_to) a user. In this case, it wouldn't make much sense for a user to
> need a foreign key associated to itself (a user belongs_to a user), so I
> don't think this is the column name you should be using. What are you
> trying to accomplish here? — Reply to this email directly or view it on
> GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/the-b-boss/blocitoff/pull/3/files#r33881173>.
>